injection/informers_test.go

```diff
@@ -61,9 +65,11 @@ func TestRegisterInformersAndSetup(t *testing.T) {
 	i.RegisterInformer(injectFooInformer)
 	i.RegisterInformer(injectBarInformer)
 
+	i.RegisterFilteredInformers(injectFooFilteredInformer)
+
 	_, infs := i.SetupInformers(context.Background(), &rest.Config{})
 
-	if want, got := 2, len(infs); got != want {
+	if want, got := 4, len(infs); got != want {
```

Comment:
Can you explain the math here in a comment?

Reply:
Fixed.